src/unitxt/standard.py

```diff
@@ -121,9 +122,69 @@ def prepare_metrics_and_postprocessors(self):
             metrics = self.metrics
         return metrics, postprocessors
 
-    def prepare(self):
+    def set_pipelines(self):
+        self.loading = SequentialOperator()
+        self.metadata = SequentialOperator()
+        self.standardization = SequentialOperator()
+        self.processing = SequentialOperator()
```

Comment:
It's not clear what is the difference between "standardization" and "processing".

Maybe "standardize_to_task", "process_task_fields"